src/features/game/types/garbage.ts

```diff
@@ -60,3 +60,36 @@ export const GARBAGE: Record<GarbageName, Garbage> = {
     sellPrice: marketRate(1),
   },
 };
+
+export type DiscardName =
```

Comment:
Are these new types necessary? You could potentially use the `isSeed` type guard in your action instead. Would prefer to not have another type that duplicates items.

```
  const isSeed = (selected: InventoryItemName): selected is SeedName =>
    isFruitSeed(selected) || selected in CROP_SEEDS();
```

Reply:
The intent was/is to have an explicit list -- like for the Sell tab -- of what is discardable.  Further, the intent (as noted in the PR description) is to provide a framework for the easy addition of other items to the discardable list.

Reply:
@Romy#7694 
How do I do this part...

```
import { CROP_SEEDS } from "features/game/types/crops";

export type DiscardName = <what goes here>;
```

I need `DiscardName` for the type for `DiscardGarbageAction`s 2nd parameter.